### DISCUSSION TITLE
Where may I start my journey with your innovation?

### DISCUSSION THREAD

**Discussion by mamunahmed-m360ict** [webcrumbs-community/webcrumbs#35] [General]
Where may I start my journey with your innovation?

**Comment by webcrumbs-community** [comment]
Hello, friend! 

Very soon we wish to have a release that you can clone and play with. 

In the meantime, would you like to be a part of it? You can contribute in so many ways: even telling us what would you like to see in a platform like ours. 

Cheers!
